test/generator/template_values_test.rb

```diff
@@ -2,26 +2,44 @@
 
 module Generator
   class TestCasesValuesTest < Minitest::Test
+    def setup
+      @arguments = {
+        abbreviated_commit_hash: nil, version: nil, exercise_name: nil, test_cases: nil
+      }
```

Comment:
I didn't do it to avoid typing, I did it so that the argument in the particular test stands out. I think making the tests more readable is important.